Accessibility/Accessibility/Accesibility.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Accesibility.swift
+//  Accessibility
+//
+//  Created by Nicole Souvenir on 1/23/19.
+//  Copyright © 2019 Nicole Souvenir. All rights reserved.
+//
+
+import Foundation
```

Comment:
You can remove this now that you have UIKit

Reply:
Updated in accessibility-5 branch.